src/components/crossDevice/CrossDeviceLink/index.js

```diff
@@ -38,6 +38,24 @@ const SECURE_LINK_VIEWS = [
   },
 ]
 
+const configHasInvalidViewIds = (viewIdsInConfig) => {
+  const validViewIds = SECURE_LINK_VIEWS.map(({ id }) => id)
+  const invalidViewIds = viewIdsInConfig.filter(
+    (viewId) => !validViewIds.includes(viewId)
+  )
+  if (invalidViewIds.length > 0) {
+    console.log(
+      'WARNING - Invalid properties in _crossDeviceLinkMethods option:',
+      invalidViewIds.join(', ')
+    )
+    console.log(
+      '_crossDeviceLinkMethods must be an array with at least 1 of the following option: "qr_code", "copy_link", "sms"'
+    )
```

Comment:
If these are warnings, should they be `console.warn` instead?
Or perhaps should they simply `throw`, since it'd be an error they can notice and recover from during development?
I can't imagine a scenario where it'd work in development but throw in production 🤔

Reply:
You're right. I originally tried to use `console.warn` and even `console.error` but they were not showing up on the browser console. Tried again today & turns out it was because of the console's text filter having some text. 🤦‍♀️  

Am avoiding throwing an error since we don't want to break the UX for the end user if the integrator chooses to still ignore the warning during integration.